src/main/java/net/sourceforge/jwbf/core/contentRep/WatchResponse.java

```diff
@@ -0,0 +1,205 @@
+package net.sourceforge.jwbf.core.contentRep;
+
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+@JsonNaming(PropertyNamingStrategy.LowerCaseStrategy.class)
+//@JsonInclude(Include.NON_DEFAULT)
+public class WatchResponse {
+	@JsonProperty("ns")
```

Comment:
Can you remove the setters in this class and use a constructor or static factory?
btw: a test will be helpful, that shows that mapping works correct :wink: